lib/css/res.scss

```diff
@@ -1836,6 +1836,21 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 			content: attr(aftercontent);
 		}
 	}
+
+	&.selected a {
+		&:first-child {
+			border-right: none;
+		}
+
+		&:last-child {
+			border-left: none;
+		}
+
+		&:only-child {
```

Comment:
good point, it's been a little while since I've used some of these css selectors :grin: